package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-get-random-values",
-  "version": "1.11.0",
+  "version": "1.11.1",
```

Comment:
Please don't bump the version in the PR, I have scripts that will create a release commit, create a signed tag, publish to npm, and create a release on GitHub automatically...

Reply:
Fixed!